src/lib/y2partitioner/dialogs/import_mount_points.rb

```diff
@@ -96,12 +96,24 @@ def label
           _("Format System Volumes")
         end
 
+        def init
+          self.value = true
+
+          store
+        end
+
         def store
           controller.format_system_volumes = value
         end
 
-        def init
-          self.value = true
+        def opt
+          [:notify]
+        end
+
+        def handle
+          store
```

Comment:
why it is needed? it will redraw table? please mention it in comment as it is not much obvious why it is called.

Reply:
Ok, I added the comment only in the commit description. I am going to copy it here too.

Reply:
Thanks, comment in commit is good, but doing this archeology to find reason is annoying. My simple rule is comment is code why we are doing it and comment in commit why we are changing it. So when you read in code you immediately see rationale for code and when you are interested why we change something e.g. between SPs or major releases, you will see it in blame.